README.md

```diff
@@ -149,34 +153,35 @@ type  │                    data
      format
 ```
 
-### Encrypted data formats
+### 5. Encrypted data formats
 
 When content is encrypted (in other words, "boxed") in SSB, it is provided as
 uninterpretable bytes, plus a tag that identifies which algorithm was used for
 encrypting it, such as `box` or `box2`.
 
 | Type code | Format code | Data length | Format name | Specification   |
-|-----------|-------------|-------------|-------------|-----------------|
-| 5         | 0           | Arbitrary   | box         | [private box]   |
-| 5         | 1           | Arbitrary   | box2        | [envelope spec] |
+|:---------:|:-----------:|-------------|-------------|-----------------|
+| 5         | 0           | Arbitrary   | box1        | [private box]   |
+| 5         | 1           | Arbitrary   | box2        | [private group] |
 
-### Generic data formats
+### 6. Generic data formats
 
 BFE supports encoding data types with no semantics attached to them. They are
 merely categorized into formats that represent their data type.
 
 | Type code | Format code | Data length | Format name | Specification                 |
-|-----------|-------------|-------------|-------------|-------------------------------|
-| 6         | 0           | Arbitrary   | UTF8 string | [UTF8]                        |
-| 6         | 1           | 1 byte      | Boolean     | Data byte is 0 for False, 1 for True |
-| 6         | 2           | 0 bytes     | Nil         | [null pointer]                |
-| 6         | 3           | Arbitrary   | Arbitrary bytes | N/A |
+|:---------:|:-----------:|-------------|-------------|-------------------------------|
+| 6         | 0           | Arbitrary   | string-UTF8 | [UTF8]                        |
+| 6         | 1           | 1 byte      | boolean     | Data byte is 0 for False, 1 for True |
+| 6         | 2           | 0 bytes     | nil         | [null pointer]                |
+| 6         | 3           | Arbitrary   | any-bytes   | N/A |
```

Comment:
```suggestion
| 6         | 3           | Arbitrary   | any-bytes   | Arbitrary sequence of bytes |
```

I can live with this PR being merged but at least this suggestion needs to be merged because we need a humane description of the specification.